src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -61,7 +61,8 @@ export class GridOutputComponent extends AngularDisposable implements IMimeCompo
 
 	constructor(
 		@Inject(IInstantiationService) private instantiationService: IInstantiationService,
-		@Inject(IThemeService) private readonly themeService: IThemeService
+		@Inject(IThemeService) private readonly themeService: IThemeService,
+		@INotificationService private _notificationService: INotificationService,
```

Comment:
Use the @Inject tag here (pretty sure that isn't actually strictly necessary since we're not using the angular DI stuff but better to be consistent)